### DISCUSSION TITLE
x-mask breaks wire:model.lazy in Safari

### DISCUSSION THREAD

**Discussion by cby016** [alpinejs/alpine#3629] [7. Bugs (archived)]
Using both x-mask and wire:model.lazy on a text input, breaks data syncing in Safari.

To duplicate, clone and setup a Livewire playground using this repo https://github.com/joshhanley/LivewirePlayground

```
git clone https://github.com/joshhanley/LivewirePlayground.git
cd LivewirePlayground
composer install
cp .env.example .env
php artisan key:generate
```

Include the x-mask plugin in the layouts/app.blade.php file

`<script defer src="https://cdn.jsdelivr.net/npm/@alpinejs/mask@3.x.x/dist/cdn.min.js"></script>`

Add the following public property in app/Http/Livewire/Main.php

`public $testValue;`

Add the following markup in the main.blade.php

```
<div x-data>
    <input type="text" x-mask="99/99/9999" wire:model.lazy="testValue">
    <br /><br />
    <div>Test Value: {{ $testValue }}</div>
</div>
```

Start the test server 
`php artisan serve`

Visit http://127.0.0.1:8000 in Safari and add a value and hit tab.  Expected behavior is to see the test value displayed underneath the text input, but nothing is displayed.  Doing the same thing in a Chromium based browser works fine.  This also seems to be an issue on iOS.

**Comment by rNAOMD** [comment]
Also facing this same issue right now.
Did you manage to solve it?

**Comment by cby016** [comment]
@rNAOMD The only work around I found was to use `wire:model.debounce.500ms` instead of `wire:model.lazy`.

**Comment by ekwoka** [comment]
Is this still an issue with LiveWire 3?
